front/.prettierrc.json

```diff
@@ -1,8 +1,7 @@
 {
-  "trailingComma": "es5",
-  "tabWidth": 4,
-  "semi": true,
-  "singleQuote": false,
-  "printWidth": 120
-
+    "trailingComma": "es5",
+    "tabWidth": 4,
+    "semi": true,
+    "singleQuote": false,
+    "printWidth": 120
```

Comment:
remove this file from git please

Reply:
wtf, why we need to remove it? 
we added it to have same code style for whole team, why why we delete it :-D

Reply:
Plz return it :)